lib/LedgerSMB/Upgrade_Tests.pm

```diff
@@ -233,6 +262,21 @@ sub _get_tests {
 
     my @tests;
 
+push @tests, __PACKAGE__->new(
+    test_query => "SELECT * FROM pg_settings
+                    WHERE name = 'client_encoding'
+                      AND setting <> 'UTF8'",
+    display_name => marktext('Client encoding must be UTF8'),
+    name => 'client_encoding',
+    display_cols => ['name','setting','unit','category','short_desc','extra_desc','context','vartype','source','min_val','max_val','enumvals','boot_val','reset_val','sourcefile','sourceline'],
+ instructions => marktext(
+                   'Please fix the client encoding'),
```

Comment:
Doesn't the client negotiate its own encoding? I mean, DBD::Pg >3.3.0 will always want to open a UTF8 encoded connection. If we require that, then why does the client encoding get set incorrectly?

Reply:
What does this have to do with the current PR?

Reply:
Indeed: what does this check have to do with the current PR? I think it does not relate to reconciliation aft all. Please remove this extra check from this PR. We might want to move it to the sanity checks later.